### PR TITLE
Add analytics meta tag component

### PR DESCRIPTION
**Pull request by edds**
Create a component which takes a links hash object from a content-store
object and turns that into a collection of meta-tags which can then be
read by our analytics JavaScript. This information in then sent to
Google as a custom dimension.

The component doesn't actually render anything visual so doesn't have a
corresponding stylesheet.

Would appreciate @dsingleton's thoughts on this.

### CODE REVIEW

File: app/views/govuk_component/docs.yml [hunk 1]
```diff
@@ -427,3 +427,26 @@
...
+  name: "Analytics Meta Tags"
+  description: "Meta tags to provide analytics information about the current page"
+  body: |
+    Currently this just takes a content-store links hash like object which it
+    can then turn into the correct analytics identifier metadata tags.
+  fixtures:
+    default: {}
+    with_orgnisations:
+      links_hash:
+        orgnisations:
```
Comment (fofr) [inline app/views/govuk_component/docs.yml]: There's some typos here. `with_orgnisations`, `orgnisations` and `lead_orgnisations`

File: app/views/govuk_component/docs.yml [hunk 2]
```diff
@@ -427,3 +427,26 @@
       next_page:
         url: "next-page"
         title: "Next page"
+- id: "analytics_meta_tags"
+  name: "Analytics Meta Tags"
+  description: "Meta tags to provide analytics information about the current page"
+  body: |
+    Currently this just takes a content-store links hash like object which it
+    can then turn into the correct analytics identifier metadata tags.
```
Comment (jamiecobbett) [inline app/views/govuk_component/docs.yml]: Is it worth providing a pointer to the code which reads the meta tags?

File: app/views/govuk_component/analytics_meta_tags.raw.html.erb [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<%
+  organisations = []
+  organisations += links_hash["organisations"] || []
```
Comment (jamiecobbett) [inline app/views/govuk_component/analytics_meta_tags.raw.html.erb]: Given that we know we want to render meta tags from attributes that aren't in the links hash, should this instead take a content item hash?

**Comment by edds** [comment]
I have updated this in relation to the feedback.

**Comment by dsingleton** [comment]
:+1: to the idea of encapsulating the analytics meta into a component. 

It feels a bit weird to be tying the component so closely to the specific `content_item` representation but presumably can be expanded to include `format`, `rendering_app`etc, without changing the calling code in the application (or slimmer).

So, generally, :+1:

**Pull request merged**